fsadmin/screenrc

```diff
@@ -0,0 +1,4 @@
+defscrollback 10000
+backtick 0 30 30 sh -c 'screen -ls | grep --color=no -o "$PPID[^[:space:]]*"'
+hardstatus string "%{= KW} %H [%`] %{= Kw}|%{-} %-Lw%{= bW}%n%f %t%{-}%+Lw"
+hardstatus alwayslastline
```

Comment:
This is completely incomprehensible. Could you add some comments explaining what this does?

Reply:
Well, "I stole it from The Internet 20 Years Ago" but ... OK, I've taken a crack at it.

This is why there's tmux in there too, for the fortunate people who don't have the screen commands burned into their muscle memory and can shift to something that has a configuration language that is not right-up-there-with-sendmail.cf in terms of unreadability.